libopae/src/token_list.c

```diff
@@ -186,6 +186,7 @@ struct _fpga_token *token_get_parent(struct _fpga_token *_t)
  * Clean up remaining entries in linked list
  * Will delete all remaining entries
  */
+/*
```

Comment:
I don't know how I feel about disabling code like this just to improve coverage.
Should we add a destructor attribute so that this gets called when the library is unloaded? I think we should be able to test it then (and get coverage).
```C
__attribute__((destructor))
```